pyproject.toml

```
@@ -0,0 +1,71 @@
+# pyproject.toml file for pywbem
+# Documentation:
+# pyproject.toml: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/
+# setuptools items: https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
+
+[build-system]
+requires = [
+    "setuptools>=66.1.0",
+    "setuptools-scm>=8.0",
+    "wheel>=0.38.1",
+]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+platforms = ["any"]
+script-files = [
+    "mof_compiler", "mof_compiler.bat"
+]
+zip-safe = true
+
+[tool.setuptools.packages.find]
+include = ["pywbem", "pywbem_mock"]
```

I suspect that you need the `*` here anyway, otherwise the "ambiguity" error is likely to happen

```suggestion
include = ["pywbem*"]
```

---

Yes, the ambiguity error did happen with `include = ["pywbem", "pywbem_mock"]`.

I tried `include = ["pywbem.*", "pywbem_mock.*"]` and that also made the ambiguity error go away. I prefer that over `pywbem*`.

---

Still risk I am afraid 😅.

These patterns in setuptools are very literal... For example, `"pywbem"` will not match `pywbem._vendor.nocasedict`. The same way `"pywbem.*"` will not match `"pywbem"`.

You probably want to have a look on the `.whl` file generated with `include = ["pywbem.*", "pywbem_mock.*"]` to see if some files are not missing.

---

Good point with the literal matching. I am now using:
`include = ["pywbem", "pywbem.*", "pywbem_mock", "pywbem_mock.*"]`
and will check the resulting wheel file.